src/components/Book.js

```diff
@@ -1,20 +1,29 @@
 import React from "react";
-import PropTypes from "prop-types";
 
 const Book = (props) => {
   const { book } = props;
   const { id, title, category } = book;
+  const { clickHandler } = props;
+  const removeBook = () => {
+    clickHandler(book);
+  };
   return (
-    <tr key={`book${id}`}>
+    <tr>
       <td>{id}</td>
       <td>{title}</td>
       <td>{category}</td>
+      <td>
+        <button
+          type="submit"
+          onClick={() => {
+            removeBook(book);
+          }}
+        >
+          Remove
+        </button>
+      </td>
```

Comment:
Please we need to validate our props by doing the `Typechecking with PropTypes`, that was the mean reason you added it in milestone 1;
![image](https://user-images.githubusercontent.com/31897434/88877145-c1c43500-d21c-11ea-8c1b-1b6452874de5.png)